etc/bento_deploy.env

```diff
@@ -53,6 +54,10 @@ BENTO_AUTHZ_DB_PASSWORD=  # TODO: SET ME WHEN DEPLOYING!
 #  - WES Client ID/secret; client within BENTOV2_AUTH_REALM
 BENTO_WES_CLIENT_ID=wes
 BENTO_WES_CLIENT_SECRET=  # TODO: SET ME WHEN DEPLOYING!
+
+#  - Grafana Client ID/secret; client within BENTOV2_AUTH_REALM
+BENTO_GRAFANA_CLIENT_ID=grafana
+BENTO_GRAFANA_CLIENT_SECRET=
```

Comment:
these two variables should also be defined in `default_config.env` to set a default value for `BENTO_GRAFANA_CLIENT_ID`